Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppProtectionPolicyiOS/MSFT_IntuneAppProtectionPolicyiOS.schema.mof

```diff
@@ -1,7 +1,8 @@
 [ClassVersion("1.0.0.0"), FriendlyName("IntuneAppProtectionPolicyiOS")]
 class MSFT_IntuneAppProtectionPolicyiOS : OMI_BaseResource
 {
-    [Key, Description("Display name of the iOS App Protection Policy.")] String DisplayName;
+    [Write, Description("Identity of the iOS App Protection Policy.")] String Identity;
+    [Write, Description("Display name of the iOS App Protection Policy.")] String DisplayName;
```

Comment:
Why are we making DisplayName Optional? I believe the new- cmdlet requires it. We should try to align where possible and always require at least one Key parameter.

Reply:
I make them optional if the displayName needs to be renamed for use case such as change of naming convention